examples/portal_advanced_security.py

```diff
@@ -0,0 +1,34 @@
+# pip install pyjwt
+import jwt
+
+from flask import Flask, request, abort
+
+app = Flask(__name__)
+
+# Somewhere in the config module
+app.config['DEPLOYMENT_1_MASTER_KEY'] = '123'
```

Comment:
@PhilippeTrounev Maybe write `deployment-1-master-key` instead of `123` here?

Reply:
yes, you can even provide a "sample key" that ends with sample. So it's same lenght as what they will be geting rwef-f23-32f2-ff-2f-3sample